r/R/array.R

```diff
@@ -84,6 +84,28 @@
 #'
 #' @rdname array
 #' @name array
+#' @examples
+#' vals <- 1:10
```

Comment:
Is `vals` here reused? I don't see it, but might be missing it in code folding... If not, it might be clearer to use `1:10` in the line below